plugins/inputs/bond/bond.go

```diff
@@ -118,10 +155,100 @@ func (bond *Bond) gatherBondPart(bondName string, rawFile string, acc telegraf.A
 	return fmt.Errorf("Couldn't find status info for '%s' ", bondName)
 }
 
+func (bond *Bond) readSysFiles(bondDir string) (sysFiles, error) {
+	/*
+		Files we may need
+		bonding/mode
+		bonding/slaves
+		bonding/ad_num_ports
+	*/
+	var err error
+	var output sysFiles
+
+	file, err := os.ReadFile(bondDir + "/bonding/mode")
+	if err != nil {
+		return sysFiles{}, fmt.Errorf("error inspecting '%s' interface: %v", bondDir+"/bonding/mode", err)
```

Comment:
```suggestion
		return sysFiles{}, fmt.Errorf("error inspecting %q interface: %v", bondDir+"/bonding/mode", err)
```
Same for the other instances.